framework/src/main/java/org/checkerframework/common/basetype/BaseTypeVisitor.java

```diff
@@ -381,42 +381,59 @@ public void processClassTree(ClassTree classTree) {
                 validateTypeOf(im);
             }
         }
+        checkExtendsImplements(classTree);
```

Comment:
The code starting at line 369 is also about extends/implements clauses. Should it be moved into `checkExtendsImplements`?